dev-util/qdevicemonitor/files/qdevicemonitor-1.0.1-qt-5.11.patch

```diff
@@ -0,0 +1,39 @@
+From d46ea62810269162ba7ceb88348c8d042c62de5c Mon Sep 17 00:00:00 2001
+From: Alexander Lopatin <alopatindev@gmail.com>
+Date: Fri, 13 Jul 2018 12:30:38 +0300
+Subject: [PATCH] fix Qt 5.11 specific issues https://bugs.gentoo.org/660932
+
+---
+ qdevicemonitor/devices/DeviceFacade.cpp | 2 +-
+ qdevicemonitor/ui/MainWindow.cpp        | 1 +
+ 2 files changed, 2 insertions(+), 1 deletion(-)
+
+diff --git a/qdevicemonitor/devices/DeviceFacade.cpp b/qdevicemonitor/devices/DeviceFacade.cpp
+index ced04b3..6df67eb 100644
+--- a/qdevicemonitor/devices/DeviceFacade.cpp
++++ b/qdevicemonitor/devices/DeviceFacade.cpp
+@@ -151,8 +151,8 @@ void DeviceFacade::loadSettings(const QSettings& s)
+     if (darkTheme.isValid())
+     {
+         m_darkTheme = darkTheme.toBool();
+-        m_colorTheme = ColorTheme::create(m_darkTheme);
+     }
++    m_colorTheme = ColorTheme::create(m_darkTheme);
```

Comment:
How is this change related to Qt 5.11?

Reply:
Oops, you are right, it's unrelated. Thanks for noticing!

It fixes a critical bug though (a crash that happens for users with freshly installed application). It appeared in [this commit](https://github.com/alopatindev/qdevicemonitor/commit/8ff2006021e812530fcc1dd7a8faded9cbfe7040).

Should I rename the commit message, a patch filename and remove `-r1`?

Reply:
Ah, not a great example of git use for atomic fixes, otherwise I would have recommended to directly use the upstream commit. Still, please move the runtime fix to a separate patch file and include a short description with a link to a bug (if there is one) for reference so we know why we are doing this. the same is not required for the trivial qt-5.11 fix (was this submitted upstream though?).

The runtime fix indeed makes a revbump necessary.

Reply:
Both fixes are submitted to upstream but I've also split them into two commits. I don't have any reported bugs though.

I've split the patch and updated ebuild. Please recheck. Thanks!